descriptors/ModuleDescriptor-template.json

```diff
@@ -229,6 +241,24 @@
         "voucher.all"
       ],
       "visible": false
+    },
+    {
+      "permissionName": "voucher.all",
+      "displayName": "Invoice module - all Voucher permissions",
+      "description": "Entire set of permissions needed to use Vouchers",
+      "subPermissions": [
+        "voucher.vouchers.item.get"
+      ],
+      "visible": false
+    },    
+    {
+      "permissionName": "voucher.all",
```

Comment:
duplicate permissionName, please add it as a sub permission to the existing permission Name

Reply:
Done